ci/docker/debug-cuda-scalapack.yaml

```diff
@@ -33,7 +33,11 @@ spack:
     mpich:
       require:
         - '+fortran'
+    fmt:
+      require:
+        - '@10'
```

Comment:
I've added a conflict upstream: https://github.com/spack/spack/pull/46280. The conflict mentions https://github.com/pika-org/pika/pull/1074.

Reply:
The conflict is merged and I've updated the spack sha here. I've removed the explicit version constraint on this PR again.